index.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css" type="text/css">
     <script src="https://kit.fontawesome.com/ba5ff8c0a6.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap" rel="stylesheet">
```

Comment:
I think Adam was supposed to do this, right? and also background, colors and etc

Reply:
yes i know, maybe i can take that out when he's added his, as i said before i just added a few bits in body to see better.